pkg/restapi/discovery/controller.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package discovery
```

Comment:
discovery should probably be in separate package, not under restapi

Reply:
done